saleor/graphql/product/types/digital_contents.py

```diff
@@ -4,17 +4,21 @@
 from ....core.permissions import ProductPermissions
 from ....product import models
 from ...core.connection import CountableDjangoObjectType
+from ...core.scalars import UUID
 from ...decorators import permission_required
 from ...meta.deprecated.resolvers import resolve_meta, resolve_private_meta
 from ...meta.types import ObjectWithMetadata
 
 
 class DigitalContentUrl(CountableDjangoObjectType):
     url = graphene.String(description="URL for digital content.")
+    token = graphene.Field(
```

Comment:
@maarcingebala If we use toke as a custom field, should we add resolver for this field?

Reply:
This only overwrites the default UUID field to our UUID which is default UUID with our errors, so probably not.

Reply:
If it works without a custom resolver then we don't need it.